packages/migration/migrations/openhim/20230328101901-potential-duplicate.cjs

```diff
@@ -0,0 +1,82 @@
+/*
+This Source Code Form is subject to the terms of the Mozilla Public
+License, v. 2.0. If a copy of the MPL was not distributed with this
+file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+OpenCRVS is also distributed under the terms of the Civil Registration
+& Healthcare Disclaimer located at http://opencrvs.org/license.
+
+Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+graphic logo are (registered/a) trademark(s) of Plan International.
+*/
+const {
+  addRouteToChannel,
+  removeRouteFromChannel,
+  upsertChannel,
+  removeChannel,
+  newChannelTemplate
+} = require('../../utils/openhim-helpers.cjs')
+
+
+const eventMarkAsDuplicateChannel = {
+  ...newChannelTemplate,
+  routes: [
+    {
+      name: 'Metrics -> Marked as duplicate',
+      secured: false,
+      host: 'metrics',
+      port: 1050,
+      path: '',
+      pathTransform: '',
+      primary: false,
+      username: '',
+      password: '',
+      forwardAuthHeader: true,
+      status: 'enabled',
+      type: 'http'
```

Comment:
@sifulovi There is a routeTemplate as well similar to the newChannelTemplate. Let's use that. And as this is the only route in this channel, let's set primary to true